velox/core/QueryCtx.h

```diff
@@ -142,8 +142,8 @@ class QueryCtx : public Context {
     if (!pool_) {
```

Comment:
```
if (pool_ == nullptr) {
   return;
}
...
```

Reply:
Thanks for finding out. I am going to fix this plus the other issue in https://github.com/facebookincubator/velox/pull/3815 which is required by Meta internal use case.